capstone/src/main/java/com/google/step/servlets/RestaurantServlet.java

```diff
@@ -91,21 +81,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Hard coded while we don't have Maps API functionality
     GeoPt geoPoint = new GeoPt((float) 42.23422, (float) -87.234987);
 
-    Entity restaurantInfo = new Entity("RestaurantInfo", id);
-    restaurantInfo.setProperty("restaurantKey", id);
-    restaurantInfo.setProperty("name", name);
-    restaurantInfo.setProperty("email", email);
-    restaurantInfo.setProperty("location", geoPoint);
-    restaurantInfo.setProperty("story", story);
-    restaurantInfo.setProperty("cuisine", cuisineList);
-    restaurantInfo.setProperty("phone", phone);
-    restaurantInfo.setProperty("website", website);
+    // The following value is hardcoded while we implement the properties.
+    String status = "OKAY";
 
-    // Both of the following values are hardcoded while we implement the properties.
-    restaurantInfo.setProperty("score", 2.5);
-    restaurantInfo.setProperty("status", "OKAY");
+    Restaurant restaurant =
```

Comment:
Wouldn't this method of obtaining a restaurant ID result in collisions of restaurant IDs in the case where a user has more than one restaurant? Which would mean that the ID wouldn't be a unique identifier?

Reply:
Updated with different ID method